Advanced Character Search/common/scripted_effects/acs_error_suppression_effects.txt

```diff
@@ -0,0 +1,19 @@
+﻿acs_suppress_error_effect = {
```

Comment:
can you just add prefixes to name of files and on script effects. I know that I was not consistent with it at beggining, but later I realized that I had a problem to find out what was an effect and what was a value for example.

So for script effects I think I used acs_se prefix.

I not sure if I have had an event but if not use acs_evt prefix.